From b8a06681c876181df7a2bd812a14860afd43c490 Mon Sep 17 00:00:00 2001
From: Liquor <lirui130@huawei.com>
Date: Fri, 18 Dec 2020 16:17:33 +0800
Subject: [PATCH] add skip_if_no_datefudge to testpkcs11.sh

Reason: test certificates in tests/testpkcs11-certs have expired
issue:https://gitlab.com/gnutls/gnutls/-/issues/1135
---
 tests/testpkcs11.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/testpkcs11.sh b/tests/testpkcs11.sh
index 9458af2..71f4e31 100755
--- a/tests/testpkcs11.sh
+++ b/tests/testpkcs11.sh
@@ -69,6 +69,8 @@ SERV="${SERV} -q"
 
 . ${srcdir}/scripts/common.sh
 
+skip_if_no_datefudge
+
 rm -f "${LOGFILE}"
 
 exit_error () {
-- 
2.27.0

